src/components/Calculator.js

```diff
@@ -47,4 +53,8 @@ const Calculator = () => (
   </div>
 );
 
+Display.propTypes = {
+  output: PropTypes.string.isRequired,
+};
+
```

Comment:
- [ ] [OPTIONAL] The proptype check is failing as currently you are passing number and not a string which you have mentioned here, please fix it :+1: